### PR TITLE
More Role API Methods

### PR DESCRIPTION
**Pull request by xNexus-ACS**

### CODE REVIEW

File: Exiled.API/Features/Roles/Scp173Role.cs [hunk 1]
```diff
@@ -247,6 +258,23 @@ public GameObject Tantrum(bool failIfObserved = false, float cooldown = 0)
         /// <param name="targetPos">The Target Position.</param>
         public void Blink(Vector3 targetPos) => BlinkTimer.ServerBlink(targetPos);
 
+        /// <summary>
+        /// Snap a <see cref="Player"/> (Attack).
+        /// </summary>
+        /// <param name="player">The <see cref="Player"/>to snap.</param>
+        public void Snap(Player player)
```
Comment (joker-119) [inline Exiled.API/Features/Roles/Scp173Role.cs:265]: If we are going to have an Attack() method for every SCP, why are we not defining it as an abstract void in an IScpRole interface or such, that way it's called the same thing in all of them?

Comment (xNexus-ACS) [reply Exiled.API/Features/Roles/Scp173Role.cs:265]: Uhm good question, I didn't thonk about it, gonna do that then

Comment (xNexus-ACS) [reply Exiled.API/Features/Roles/Scp173Role.cs:265]: working on it

**Comment by xNexus-ACS** [comment]
not finished, I need to test the attack method in 939, 0492 & 096

**Review by louis1706 — APPROVED** [review]

File: Exiled.API/Features/Roles/Scp106Role.cs [hunk 2]
```diff
@@ -82,7 +82,7 @@ internal Scp106Role(Scp106GameRole baseRole)
         /// <summary>
         /// Gets the <see cref="Scp106Attack"/>.
         /// </summary>
-        public Scp106Attack Attack { get; }
+        public Scp106Attack AttackAbility { get; }
```
Comment (louis1706) [inline Exiled.API/Features/Roles/Scp106Role.cs:85]: There a breaking change here

**Review by louis1706 — CHANGES_REQUESTED** [review]

File: Exiled.API/Features/Roles/Scp106Role.cs (hunk 2 above)
Comment (xNexus-ACS) [reply Exiled.API/Features/Roles/Scp106Role.cs:85]: it makes conflict with the Attack Method

Comment (xNexus-ACS) [reply Exiled.API/Features/Roles/Scp106Role.cs:85]: that's why I changed it

**Review by louis1706 — APPROVED** [review]
BreakChange

**Pull request closed without merging**